src/commands/dev/lib/dev-router.ts

```diff
@@ -53,20 +58,20 @@ export default async function(
         continue;
       }
 
-      if (!src.startsWith('^')) {
-        src = `^${src}`;
-      }
-
-      if (!src.endsWith('$')) {
-        src = `${src}$`;
-      }
+      src = src.replace(/^\^?\/?(.*)$/, (_, path) => {
```

Comment:
I'm not a huge fan of regex so I think I would prefer how it was written before but with a check for `src.startsWith('/')`